tensorflow/cc/gradients/nn_grad_test.cc

```diff
@@ -191,5 +191,13 @@ TEST_F(NNGradTest, MaxPoolGradV2Helper) {
   RunTest(x, x_init_value, y, y_shape);
 }
 
+TEST_F(NNGradTest, LRN){
+  TensorShape x_shape({10, 10, 54, 10});
```

Comment:
Nit: Since the shapes are the same you can just have one shape variables and pass it twice to the Run call.